client/src/components/solution-display-widget/index.tsx

```diff
@@ -70,10 +74,13 @@ export function SolutionDisplayWidget({
       bsStyle='primary'
       className='btn-invert'
       data-cy={dataCy}
+      title='view code button'
```

Comment:
Adding a title will add a tooltip, if we are going to keep it around, we need it to be descriptive of what the button for, like `navigate to solution of this certifcation` and it needs to be translatable.

we can cross that bridge another time, so here is my suggestion

```suggestion
```

If you want to add it, you can add descriptive title in `.json` file and use it as a value for the title. `viewText` is a good example